grpc/src/proto.proto

```diff
@@ -25,10 +25,18 @@ service Octree {
   // TODO(sirver): This is not the most efficient interface for a gRPC
   // version. We probably just send the node metadata to the client and let
   // the client do figure out GetVisibleNodes.
+  rpc GetRootBoundingCube(Empty) returns (GetRootBoundingCubeReply);
   rpc GetVisibleNodes(GetVisibleNodesRequest) returns (GetVisibleNodesReply);
   rpc GetNodeData(GetNodeDataRequest) returns (GetNodeDataReply);
 }
 
+message Empty {
```

Comment:
Just call it GetRootBoundingCubeRequest. We might want to add stuff there in the future.